libs/dh/market-participant/actors/feature-actors/tsconfig.lib.json

```
@@ -13,5 +13,8 @@
     "jest.config.ts",
     "src/**/*.test.ts"
   ],
-  "include": ["src/**/*.ts"]
+  "include": [
+    "src/**/*.ts",
+    "../feature-delegation/src/lib/dh-delegation-tab.component.ts"
+  ]
```

nitpick: probably added automatically by VS Code after file was moved to another location

```suggestion
  "include": ["src/**/*.ts"]
```